canonical-web-and-design/vanilla-framework
# Add interactive chips

## Done

Adds interactive chips (using `button` elements). Also makes sure chips respect aria-pressed (#3416)
Drive-by: fix chips overrides in search&filter component.

Fixes #4167
Fixes #3416 
Fixes #4166

## QA

- Open [demo](https://vanilla-framework-4169.demos.haus/docs/examples/patterns/chip/interactive)
- Check if new examples with interactive chips work:
  - https://vanilla-framework-4169.demos.haus/docs/examples/patterns/chip/interactive
  - All variants (interactive are at the end of each row): https://vanilla-framework-4169.demos.haus/docs/examples/patterns/chip/variants
- Make sure search&filter looks as expected (note that Vanilla examples are not fully interactive, just expand/collapse):
  - https://vanilla-framework-4169.demos.haus/docs/examples/patterns/search-and-filter/with-chips
- Review updated documentation:
  - https://vanilla-framework-4169.demos.haus/docs/patterns/chip


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
